ios/Classes/PTFlutterViewController.h

```diff
@@ -14,6 +14,21 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic) BOOL needsRemoteDocumentLoaded;
 @property (nonatomic) BOOL documentLoaded;
 
+// viewer options
+@property (nonatomic, assign, getter=isReadOnly, setter=setReadOnly:) BOOL readOnly;
```

Comment:
I believe `setReadOnly:` should be the default and does not need to be specified. Same with `setThumbnailViewEditingEnabled:`, `setContinuousAnnotationEditing` below.